src/index.html

```diff
@@ -11,6 +11,35 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    action="/"
+    method="get"
+    data-qa="big"
+    >
```

Comment:
following code styling 
```suggestion
    <form
      action="/"
      method="get"
      data-qa="big"
    >
```